src/test/java/org/fundacionjala/coding/wilson/KataFizzBuzzTest.java

```diff
@@ -0,0 +1,78 @@
+package org.fundacionjala.coding.wilson;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class KataFizzBuzzTest {
+    /**
+     *
+     */
+    @Test
+    public void numberTestFizzBuzz() {
+        final int num = 15;
+        assertEquals("FizzBuzz", KataFizzBuzz.sendString(num));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void numberTestFizz() {
+        final int num = 3;
+        assertEquals("Fizz", KataFizzBuzz.sendString(num));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void numberTestBuzz() {
+        final int num = 5;
+        assertEquals("Buzz", KataFizzBuzz.sendString(num));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void convertNumberTestBuzz() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "numberTestBuzz" on line 33. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)